docker-compose.yml

```diff
@@ -19,11 +19,15 @@ services:
     command: bash -l -c "npm install && npm start"
     environment:
       - DATABASE_URL=postgresql://postgres@db/app
+      - SMTP_HOST=mail
+      - SMTP_PORT=1025
+      - SMTP_USER=
+      - SMTP_PASS=
```

Comment:
I've updated the environment variables for email to be more configurable, rather than hard-coded to Gmail. I've also set the defaults to connect to the Mailcatcher development SMTP server that was already configured in our Docker Compose config.